keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -2404,7 +2404,14 @@ in back-to-back releases.
 - support the list of features mentioned under phrase 2
 
 ## GA
-- complete type check for CRD and aggregated types
+- Complete type check for CRD and aggregated types
+- Scalability evaluation. Evaluate ValidatingAdmissionPolicy scalability including how many validators it could run, runtime cost evaluation, evaluating the reasonable ResourceQuota for
+  ValidatingAdmissionPolicy, how much faster comparing with Webhook, the scale target, etc.
+- Fix the known issues along the way([variables should keep the type info](https://github.com/kubernetes/kubernetes/issues/120239), [variables should mark omitempty](https://github.com/kubernetes/kubernetes/issues/120491), [properly escape](https://github.com/kubernetes/kubernetes/issues/118230))
```

Comment:
https://github.com/kubernetes/kubernetes/issues/122658 also?

Reply:
Sure. I have the most updated issue listed marked and tracked here as well: https://github.com/kubernetes/kubernetes/issues/122936